### PR TITLE
upgrading the postgres db for external cms's

### PR DESCRIPTION
**Pull request by Matt-Bishop**
https://www.pivotaltracker.com/story/show/137563973

### CODE REVIEW

File: lib/client_app_provision_modules/heroku_addons.rb [hunk 1]
```diff
@@ -1,9 +1,12 @@
 module ClientAppProvisionModules
+
   class HerokuAddons < ClientAppProvisionModules::Base
+    include G5InternalOverrides
     def provision
       Rails.logger.info "Adding addons to #{client_app.name}..."
       defaults["addons"].each do |addon|
         begin
+          addon = g5_internal_check(client_app, addon)
```
Comment (variousred) [inline lib/client_app_provision_modules/heroku_addons.rb]: i believe this will eliminate ALL the other addons

Comment (variousred) [inline lib/client_app_provision_modules/heroku_addons.rb]: oh i see if its not internal and the hash key doesnt match it passes through the addon name

Comment (variousred) [inline lib/client_app_provision_modules/heroku_addons.rb]: though I might call that method `upgrades_pg_if_internal`

Comment (Matt-Bishop) [inline lib/client_app_provision_modules/heroku_addons.rb]: Seems to work fine for me - We'll chat tomorrow

Comment (Matt-Bishop) [inline lib/client_app_provision_modules/heroku_addons.rb]: well- in theory one day we may want to add other addons to the list  for internal only apps.

**Pull request merged**